toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1001 @@
+    </message>
+</context>
+<context>
+    <name>FingerTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Invert</source>
+        <translation>Invertieren</translation>
+    </message>
+</context>
+<context>
+    <name>FullColorBrushTool</name>
+    <message>
+        <source>Thickness</source>
+        <translation>Dicke</translation>
+    </message>
+    <message>
+        <source>Pressure Sensitivity</source>
+        <translation>Druck</translation>
+    </message>
+    <message>
+        <source>Opacity:</source>
+        <translation>Opazität:</translation>
+    </message>
+    <message>
+        <source>Hardness:</source>
+        <translation>Härte:</translation>
+    </message>
+    <message>
+        <source>Preset:</source>
+        <translation>Vorlage:</translation>
+    </message>
+</context>
+<context>
+    <name>FullColorEraserTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Opacity:</source>
+        <translation>Opazität:</translation>
+    </message>
+    <message>
+        <source>Hardness:</source>
+        <translation>Härte:</translation>
+    </message>
+    <message>
+        <source>Type:</source>
+        <translation>Typ:</translation>
+    </message>
+    <message>
+        <source>Invert</source>
+        <translation>Invertieren</translation>
+    </message>
+    <message>
+        <source>Frame Range</source>
+        <translation>Frame-Reichweite</translation>
+    </message>
+</context>
+<context>
+    <name>HookTool</name>
+    <message>
+        <source>Snap</source>
+        <translation>Einrasten</translation>
+    </message>
+</context>
+<context>
+    <name>MagnetTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+</context>
+<context>
+    <name>PaintBrushTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Mode:</source>
+        <translation>Modus:</translation>
+    </message>
+    <message>
+        <source>Selective</source>
+        <translation>Selektiv</translation>
+    </message>
+</context>
+<context>
+    <name>PinchTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Corner:</source>
+        <translation>Ecke</translation>
+    </message>
+    <message>
+        <source>Manual</source>
+        <translation>Manuell</translation>
+    </message>
+</context>
+<context>
+    <name>PlasticTool</name>
+    <message>
+        <source>Build Skeleton</source>
+        <translation>Skelett aufbauen</translation>
+    </message>
+    <message>
+        <source>Paint Rigid</source>
+        <translation>Rigide bemalen</translation>
```

Comment:
When I have the program running I can get the exact references. For instance I don't know if the English version's "Skeleton" is correct, since the Japanese uses Bone and from my experience in Blender there are bones for rigging. So I'm sometimes left guessing as to what the specific tool does in order to get a better translation.